README.md

```diff
@@ -1,3 +1,26 @@
+# Book Store
+
+> Book Store is a website that allows you to add, remove and view books.
+
+## Built With
+
+- ReactJS
+- Redux
+- React Router
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+## Authors
+
+👤 **Ezekiel Utshudi**
+
+- GitHub: [@EzekielUtshudi](https://github.com/EzekielUtshudi)
+- Twitter: [@UtshudiEzekiel](https://twitter.com/UtshudiEzekiel)
+- LinkedIn: [@UtshudiEzekiel](https://www.linkedin.com/in/ezekiel-utshudi-195782162/)
+
+
 # Getting Started with Create React App
```

Comment:
- [ ] Currently, You have two `"Getting started"` sections in the readme file. Please remove the first one and the `"Author section" `should be placed after the `"Getting started"` section. :+1: 

- [ ] Please improve the `"Getting Started"` section. This should be an example of how you may give instructions on setting up your project locally.

Reply:
Thank you